code/modules/paperwork/pen.dm

```diff
@@ -64,6 +65,43 @@
 	to_chat(user, "<span class='notice'>\The [src] will now write in [colour].</span>")
 	desc = "It's a fancy four-color ink pen, set to [colour]."
 
+/obj/item/weapon/pen/fountain
+	name = "fountain pen"
+	desc = "It's a common fountain pen, with a faux wood body."
+	icon_state = "pen-fountain"
+	font = FOUNTAIN_PEN_FONT
+
+/obj/item/weapon/pen/fountain/captain
+	name = "captain's fountain pen"
+	desc = "It's an expensive Oak fountain pen. The nib is quite sharp."
+	icon_state = "pen-fountain-o"
+	force = 5
+	throwforce = 5
+	throw_speed = 4
+	colour = "crimson"
+	materials = list(MAT_GOLD = 750)
+	sharpness = IS_SHARP
+	resistance_flags = FIRE_PROOF
+	var/unique_reskin = 1
+	var/list/skins = list("Oak" = "pen-fountain-o", "Gold" = "pen-fountain-g", "Rosewood" = "pen-fountain-r", "Black and Silver" = "pen-fountain-b","Command Blue" = "pen-fountain-cb", "Cancel" = null)
+
+/obj/item/weapon/pen/fountain/captain/attack_self(mob/living/carbon/user)
+	if(unique_reskin)

```

Comment:
The unique reskin should probably be in Alt Click, it will be in line with guns that allow reskins.